test/drenv/ramen.py

```diff
@@ -53,7 +53,7 @@ def dump_e2e_config(env):
         with open(path, "w") as f:
             f.write(data)
 
-        e2e_config["Clusters"][e2e_name] = {"kubeconfigpath": path}
+        e2e_config["Clusters"][e2e_name] = {"name": e2e_name, "kubeconfigpath": path}
```

Comment:
e2e_name is wrong (e.g. "c1") - we ned the cluster_name (e.g. "dr1").

Please show the generated config with the change in the commit message, this helps to understand the change, and will quickly show error like this.